components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -441,7 +457,7 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
         setBodyText("");
         setHighlightLink("");
         setDate(undefined);
-        setIsDivFocused(false);
+        setDialogOpen(false);
```

Comment:
These changes are good like we discussed in our pairing session yesterday, but they are not related to the fix.

Create a separate PR for moving `setIsDivFocused` -> `setDialogOpen` and `isDivFocused` -> `dialogOpen`.